Lib/datetime.py

```diff
@@ -925,7 +925,33 @@ def ctime(self):
             self._day, self._year)
 
     def strftime(self, fmt):
-        "Format using strftime()."
+        """strftime(fmt) -> date_string
```

Comment:
The "input -> output" style of doc-strings is only used in functions and class-methods in this module, but not for normal (instance) methods. I don't see that it adds anything in this case (and the others changed by this PR).